### PR TITLE
Schedule Proposal

### PR DESCRIPTION
**Pull request by daviddias**
Me and @whyrusleeping went ahead and prepared a schedule proposal for the IPFS Workshop, to give some structure and define expectations for this event.

The main idea with the proposed schedule is getting everyone up to speed with the current endeavours on the IPFS project during the mornings and hands on/hacking session on the afternoons where the leads of the project can guide the attendees on how to use each project and discuss its roadmap and where the community can contribute.

On the second day, we also have time allocated for community presentations (for anyone that wants to volunteer) and a very fun session, `idea lightround`, where everyone gets 2 minutes to present a project they want to develop with IPFS.

**Next:**
- [ ] Review the proposal (@jbenet @em-ly)
- [ ] Create an issue for each of the sessions with description, goal and outcomes
- [ ] Accept proposals for the free sessions on Day 2

### CODE REVIEW

**Comment by daviddias** [comment]
em-ly says: LGTM! Love the idea of the idea lightening round.@jbenet can you please review and sign off? - from a train with love xo

**Comment by RichardLitt** [comment]
LGTM.

**Comment by haadcode** [comment]
Would it make sense to go through the most important projects, the ones we'd like people to contribute to, in a rapid fire style like we did the other day?

**Comment by daviddias** [comment]
@haadcode Good idea. I believe we will have some time left from the time alocated for `FREE SESSIONS`, we can use that for project speedround

**Comment by ghost** [comment]
I'd like to split my infrastructure talk into "ipfs.io gateway infrastructure", and "connectivity".

**Comment by daviddias** [comment]
@lgierth for sure. How much time do you want for each?

**Comment by ghost** [comment]
@diasdavid same as now, just split it in half. The gateway infrastructure part will be maybe 3 minutes and then room for questions, and connectivity will be just a teaser for packet switching :)

File: SCHEDULE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+| 10:00h | --       | people arrive and set up   | --
+| 10:15h | talk     | welcome presentation       | Juan or David
+| 10:30h | talk     | The State of IPFS          | Juan
+| 11:30h | talk     | IPLD                       | Juan
+| 12:15h | talk     | libp2p                     | David and Jeromy
+| 13:00h | open     | Lunch Break                | --
+| 14:15h | workshop | go-ipfs hands on           | Jeromy
+| 15:30h | workshop | IPLD hands on              | Juan and Friedel(remote)
+| 16:45h | workshop | libp2p hands on            | David and Jeromy
+| 18:00h | --       | end of day 1               | --
```
Comment (jbenet) [inline SCHEDULE.md]: I think the workshop should be about doing work, not presenting. I think there's way too many presentations. 

I hoped this event was more creating space for a lot of IPFS folks to get together and work on things together that require people's attention -- having important discussions for development. Not intro talks.

There's a ton of things that have been brought up in the last few days to be discussed during the workshop. Let's do that instead.

**Comment by jbenet** [comment]
My feedback: More working on important problems together, less presentations. I think this workshop should be sync development/discussions, not intro presentations??

**Comment by jbenet** [comment]
+1 for rapid fire showcase

**Comment by daviddias** [comment]
Updated the schedule based on @jbenet feedback. Going to merge and open issues for the sessions to be had (for problem solving) during the workshop

**Comment by jbenet** [comment]
:+1:

**Pull request merged**